src/system/ThemeModule/Engine/Asset/JsResolver.php

```diff
@@ -39,19 +38,19 @@ class JsResolver implements ResolverInterface
     private $combine;
 
     public function __construct(
-        ZikulaHttpKernelInterface $kernel,
+        string $env,
         AssetBag $bag,
         MergerInterface $merger,
         bool $combine = false
     ) {
-        $this->kernel = $kernel;
         $this->bag = $bag;
         $this->merger = $merger;
-        $this->combine = 'prod' === $this->kernel->getEnvironment() && $combine;
+        $this->combine = ('prod' === $env) && $combine;
```

Comment:
cs: please remove the unneeded parenthesis again

Reply:
in this case I prefer them.  a === b && c is confusing to look at.